cloudtasks/api/TasksSample/CreateHttpTask.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START cloud_tasks_create_http_task]
+
+using Google.Cloud.Tasks.V2Beta3;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using System;
+
+class CreateHttpTask
+{
+    public string CreateTask(
+        string projectId = "YOUR-PROJECT-ID",
+        string location = "YOUR-LOCATION-ID",
```

Comment:
Is the location id usually something like "us-west1"?  In that case, I think we'll provide a better clue to the reader what the value should be if we show the value "us-west1" or the like instead of the more abstract "YOUR-LOCATION-ID".